### ISSUE TITLE
Separate out the theme

### ISSUE THREAD

**Issue by ibnishak** [legostax/dotfiles#20]
If you could separate out and release the wibar theme, that would be awesome

**Comment by legostax** [comment]
My code is quite a mess. My apologies for that. I started making this when I knew practically nothing. When you say 'release the wibar theme' do you mean separately as its own widget?

I need to get Github to send me an email every time a new issue is opened on here, my bad.

**Comment by ibnishak** [comment]
> When you say 'release the wibar theme' do you mean separately as its own widget?
Not necessarily. If you could split the rc.lua to modular chunks so that wibar related code is a separate file, that would help those who want that part immensely. 

For eg, the way empressnoodle split the lua code to modular pieces here: https://github.com/elenapan/dotfiles/tree/master/config/awesome

> I need to get Github to send me an email every time a new issue is opened on here, my bad.

No issue. Take your time. Wonderful work you got there. :star2:

**Comment by legostax** [comment]
In all honesty, I plan on reprogramming this entire repo from the ground up this summer (as is evident by the other issues and the v2.0 branch). I will make sure this is a high priority in my new design.

Between my job, school, and freelance work, I simply don't have the time right now to be working on this to make sure it is actually good quality and not something thrown together. After this semester finishes in a few weeks, I'll be getting to work on it then.

I have marked this as an enhancement for 2.0. Thank you :)

**Comment by ibnishak** [comment]
Looking forward!!
